Components/ListIcons/_style.scss

```diff
@@ -34,39 +34,69 @@
       @include grid(4,
       $gutter-vertical: ($gutter-width * 2));
     }
+  }
 
-    .item {
-      display: flex;
-      flex-direction: column;
-      max-width: 320px;
+  .feather {
+    height: auto;
+    width: 100%;
+  }
 
-      &-icon {
-        height: 45px;
-        margin-bottom: 15px;
-        width: 45px;
+  .item {
+    display: flex;
+    flex-direction: column;
+    max-width: 320px;
 
-        svg {
-          height: auto;
-          width: 100%;
-        }
-      }
+    &-icon {
+      height: 45px;
+      margin-bottom: 15px;
+      width: 45px;
+    }
+
+    &-content {
+      @include reset-content-margins;
+
+      flex-grow: 1;
+    }
 
-      &-content {
-        @include reset-content-margins;
+    &-link {
+      padding-bottom: 0;
+      padding-top: 0;
+      text-decoration: none;
 
-        flex-grow: 1;
+      &::before {
+        bottom: 0;
       }
+    }
 
-      &-link {
-        padding-bottom: 0;
-        padding-top: 0;
-        text-decoration: none;
+    &--centered {
+      text-align: center;
+
+      .item-icon {
+        align-items: center;
+        background-color: $color-background-hero;
+        border-radius: 50%;
+        display: flex;
+        height: 80px;
+        margin-left: auto;
```

Comment:
Idea: As flexbox is used for vertical centering, it could use justify-content for horizontal centering as well, instead of margin auto.

Reply:
That just made me realise, the whole row should probably then be centered in center mode?
![Screen Shot 2019-11-08 at 10 44 18](https://user-images.githubusercontent.com/2886550/68466485-cc651700-0214-11ea-9399-fc113004604f.jpg)